### PR TITLE
fix: watcher emit data

### PR DESCRIPTION
**Pull request by o-rumiantsev**
Closes: https://github.com/o-rumiantsev/Dirsync/issues/9

### CODE REVIEW

**Comment by o-rumiantsev** [comment]
Blocked by https://github.com/o-rumiantsev/Dirsync/pull/10

File: lib/watcher/create-watcher.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 const fs = require('fs');
 const path = require('path');
 const EventEmitter = require('events');
+const { sequential } = require('metasync');
```
Comment (mille-nium) [inline lib/watcher/create-watcher.js]: Unnecessary require statement

File: test/watcher.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+        Buffer.from(''),
+      ]
+    ]);
+
+    watcher.stop();
+    test.end();
+  });
+});
+
+watcherTest.test('Wather handle file change', test => {
```
Comment (mille-nium) [inline test/watcher.js]: `Watcher`

File: test/watcher.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        '/home/o-rumiantsev/Github/Dirsync/test/fixtures/watcher/file.ext',
+      ],
+    ]);
+
+    watcher.stop();
+    test.end();
+  });
+});
+
+watcherTest.test('Wathcer handle create file with initial data', test => {
```
Comment (mille-nium) [inline test/watcher.js]: `Watcher`

**Comment by mille-nium** [comment]
BTW, don't we have to write test for directory creation and deletion?

**Comment by o-rumiantsev** [comment]
> BTW, don't we have to write test for directory creation and deletion?

@mille-nium I think we can add it in separate PR

**Pull request merged**